www/codemirror/codemirror-index.html

```diff
@@ -539,8 +529,73 @@
       toggleMarkup(button, openingMarkup, closingMarkup)        
     }
 
-    const convertSelectedLinesToList = (char) => {
-      editor.replaceSelection(`${char} ${editor.getSelection().replace(/\n/gi, `\n${char} `)}\n${char} `)
+    const toggleListForSelectedLines = (char) => {
```

Comment:
this should match the 2016 wikitext editor behavior and just toggle the list setting of the current line without adding a newline, regardless of selection size

Reply:
the 2016 wikitext editor behavior adds new lines in a few cases where the selection starts after the 1st character of the line. which case are you referring to?